src/main/resources/lang/default.lang

```diff
@@ -1976,3 +1979,16 @@ types:
 	# Hooks
 	money: money
 	region: region¦s
+
+# -- Boss Bars --
+bossbar styles:
```

Comment:
You're missing the defiance of `bossbar styles` and `bossbar flags` under `types:`